File: Proyecto/02. Software/02. Back-End Java/workspace/sell.it.easy/src/main/java/com/sell/it/easy/entity/Organizacion.java
package com.sell.it.easy.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Lob;
import javax.persistence.ManyToOne;

@Entity
public class Organizacion {

	@Id
	@GeneratedValue(strategy=GenerationType.AUTO)
	private Integer id;
	
	@Column(nullable=false)
	private String CIF;
	
	@Column(nullable=false)
	private String nombre;
	
	@Column(nullable=false)
	private String direccionFiscal;
	
	@Lob
	private String imagenString;
	
	@ManyToOne(optional=false)
	private TipoProducto tipoProducto;
	
	private String telefono;
	
	public Organizacion(){}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getCIF() {
		return CIF;
	}

	public void setCIF(String cIF) {
		CIF = cIF;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getDireccionFiscal() {
		return direccionFiscal;
	}

	public void setDireccionFiscal(String direccionFiscal) {
		this.direccionFiscal = direccionFiscal;
	}

	

	public String getImagenString() {
		return imagenString;
	}

	public void setImagenString(String imagenString) {
		this.imagenString = imagenString;
	}

	public TipoProducto getTipoProducto() {
		return tipoProducto;
	}

	public void setTipoProducto(TipoProducto tipoProducto) {
		this.tipoProducto = tipoProducto;
	}

	public String getTelefono() {
		return telefono;
	}

	public void setTelefono(String telefono) {
		this.telefono = telefono;
	}
	
}

File: Proyecto/02. Software/01. Front-End iOS/sell.it.easy_0.0.2/sell.it.easy/sell.it.easy/Organizacion.swift
//
//  Organizacion.swift
//  sell.it.easy
//
//  Created by David Márquez Delgado on 15/8/17.
//  Copyright © 2017 David Márquez Delgado. All rights reserved.
//

import ObjectMapper

class Organizacion : Mappable {

    var id : Int!
    var CIF : String!
    var nombre : String!
    var direccionFiscal : String!
    var imagenString : String?
    // var tipoProducto : TipoProducto
    var telefono : String!
    
    init(id: Int, CIF: String, nombre: String, direccionFiscal: String, imagenString: String?, telefono: String) {
        self.id = id
        self.CIF = CIF
        self.nombre = nombre
        self.direccionFiscal = direccionFiscal
        self.imagenString = imagenString
        self.telefono = telefono
    }
    
    required init?(map: Map) {}
    
    func mapping(map: Map) {
        id <- map["id"]
        CIF <- map["CIF"]
        nombre <- map["nombre"]
        direccionFiscal <- map["direccionFiscal"]
        imagenString <- map["imagenString"]
        telefono <- map["telefono"]
    }
    
    
}
